cmd/aries-agent-mobile/pkg/wrappers/command/aries.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package command
+
+import (
+	"fmt"
+
+	"github.com/hyperledger/aries-framework-go/pkg/storage/mem"
+
```

Comment:
any reason for this blank line? usually imports from `aries-framework-go` are grouped together and are the last group of imports.

Reply:
i was having issues with the linter. i think this is an oversight. will clean up 👍

Reply:
"github.com/hyperledger/aries-framework-go/pkg/storage/mem"

should go right under (with no blank lines):
"github.com/hyperledger/aries-framework-go/pkg/framework/aries"

imports are sorted alphabetically, you don't need to worry about their order, `go import` does it for you, or if you use an IDE, there should be a shortcut to auto format your files. It's much easier to let the tool handle the imports sorting.